.github/workflows/ci.yml

```diff
@@ -135,7 +135,7 @@ jobs:
       - name: Use Node.js
         uses: actions/setup-node@v4
         with:
-          node-version: 18
+          node-version-file: package.json
```

Comment:
It seems this means that it will now always use the newest LTS version (currently Node 22) instead of Node 18 like before:

```
Resolved package.json as >=18
Found in cache @ /opt/hostedtoolcache/node/22.12.0/x64
```

I would prefer to continue using the lowest supported version to ensure compatibility. And to specify it explicitly.
Maybe we could add a `.node-version` file and reference that?

Reply:
Indeed.

Reply:
I chose .nvmrc because it's a manifest that many tools support (nvm, asdf, etc...)